.vscode/tasks.json

```diff
@@ -0,0 +1,30 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
Let's drop this comment. It's not an official JSON construct and it created problems in the past (in `tslint.json`).